contracts/external/interestsetters/DoubleExponentInterestSetter.sol

```diff
@@ -119,14 +119,19 @@ contract DoubleExponentialInterestSetter is
             });
         }
 
-        uint256 result = 0;
-        uint256 polynomial = BASE;
+        // process the first coefficient
+        uint256 coefficients = s.coefficients;
+        uint256 result = uint8(coefficients) * BASE;
```

Comment:
So this is `C_0` right?

Reply:
This is special-case C_0 yeah since I want it to multiplied by X^0 (aka 1) but 0 is not a power of 2